s3fs/core.py

```diff
@@ -18,10 +17,14 @@
 
 logging.getLogger('boto3').setLevel(logging.WARNING)
 logging.getLogger('botocore').setLevel(logging.WARNING)
-S3_RETRYABLE_ERRORS = (
-    socket.timeout,
-    trans.ReadTimeoutError, trans.IncompleteReadError
-)
+
+try:
+    from boto3.s3.transfer import S3_RETRYABLE_ERRORS
+except ImportError:
+    S3_RETRYABLE_ERRORS = (
+        socket.timeout,
```

Comment:
Sadly not.  We could do more pinning in travis.yml, but the issue is still live.

I still think that transfer errors are serious (even though rare) but I think that this fix is more likely to make us aware of them more quickly.